components/fee-button.js

```diff
@@ -79,46 +79,43 @@ export default function FeeButton ({ parentId, hasImgLink, baseFee, ChildButton,
   const repetition = me ? data?.itemRepetition || 0 : 0
   const formik = useFormikContext()
   const boost = Number(formik?.values?.boost) || 0
-  const cost = baseFee * (hasImgLink ? 10 : 1) * Math.pow(10, repetition) + Number(boost)
+  const cost = baseFee * Math.pow(10, repetition) + Number(boost)
 
   useEffect(() => {
     formik?.setFieldValue('cost', cost)
   }, [formik?.getFieldProps('cost').value, cost])
 
+  const imageFees = formik?.getFieldProps('imageFees').value || { fees: 0 }
+  const totalCost = cost + imageFees.fees
+
   const show = alwaysShow || !formik?.isSubmitting
   return (
     <div className={styles.feeButton}>
-      <ActionTooltip overlayText={numWithUnits(cost, { abbreviate: false })}>
-        <ChildButton variant={variant} disabled={disabled}>{text}{cost > 1 && show && <small> {numWithUnits(cost, { abbreviate: false })}</small>}</ChildButton>
+      <ActionTooltip overlayText={numWithUnits(totalCost, { abbreviate: false })}>
+        <ChildButton variant={variant} disabled={disabled}>{text}{totalCost > 1 && show && <small> {numWithUnits(totalCost, { abbreviate: false })}</small>}</ChildButton>
       </ActionTooltip>
       {!me && <AnonInfo />}
-      {cost > baseFee && show &&
+      {totalCost > baseFee && show &&
         <Info>
-          <Receipt baseFee={baseFee} hasImgLink={hasImgLink} repetition={repetition} cost={cost} parentId={parentId} boost={boost} />
+          <Receipt baseFee={baseFee} imageFees={imageFees} repetition={repetition} cost={totalCost} parentId={parentId} boost={boost} />
         </Info>}
     </div>
   )
 }
 
-function EditReceipt ({ cost, paidSats, addImgLink, boost, parentId }) {
+function EditReceipt ({ cost, paidSats, imageFees, boost, parentId }) {
   return (
     <Table className={styles.receipt} borderless size='sm'>
       <tbody>
-        {addImgLink &&
-          <>
-            <tr>
-              <td>{numWithUnits(paidSats, { abbreviate: false })}</td>
-              <td align='right' className='font-weight-light'>{parentId ? 'reply' : 'post'} fee</td>
-            </tr>
-            <tr>
-              <td>x 10</td>
-              <td align='right' className='font-weight-light'>image/link fee</td>
-            </tr>
-            <tr>
-              <td>- {numWithUnits(paidSats, { abbreviate: false })}</td>
-              <td align='right' className='font-weight-light'>already paid</td>
-            </tr>
-          </>}
+        <tr>
+          <td>{numWithUnits(0, { abbreviate: false })}</td>
+          <td align='right' className='font-weight-light'>edit fee</td>
+        </tr>
```

Comment:
Added `0 sats edit fee` so I can use `+ X sats imageFees` ... Also, I might should not have removed the usage of `paidSats` in the edit receipt.

Reply:
Example receipts:

_reply:_

![2023-10-27-031624_509x218_scrot](https://github.com/stackernews/stacker.news/assets/27162016/73ca8d6c-0c4e-4841-a96f-0ebe6811d2a1)

_edit:_

![2023-10-27-031644_506x188_scrot](https://github.com/stackernews/stacker.news/assets/27162016/cacca0f1-90fa-494c-bbf1-53f94f59d05d)



I realized `paidSats` wasn't used before since `addImgLink` was not used.

So I think it's not a breaking UX change. I think it's fine to mention that editing itself does not cost anything but fees were added because of images.

What might be confusing though is why image suddenly cost something since they don't cost anything until 10 MB were uploaded within the last 24 hours. There is no mention of this in the receipt.

I played a bit around with a [ProgressBar](https://react-bootstrap.netlify.app/docs/components/progress/), but that also didn't feel right. Maybe mentioning it in the FAQ is enough? Or a info box in the receipt? But then we have a info box inside a info box, lol.